### ISSUE TITLE
Feature Request: Customizable Toolbar -> for AngularMaterial theme

### ISSUE THREAD

**Issue by thomi137** [fullcalendar/fullcalendar#5127]
I am looking for a way to write my own toolbar, such as if I would create a View.  It could be that I missed the way to do it, but I figure it would be in your docs if there was an obvious way to do it. 

I switched from Bootstrap to Angular Material, mainly for technical reasons, and would like to have the look and feel of it on fc. Of course, I managed to get the buttons right and I wrote something similar to the bootstrap plugin for having the proper styles in the toolbar (still fighting with  the ripple effect...).  Also, the way your code works does not really work with material icons, so I would very much like to add some directives, e.g. to a MatToolbar. At the moment, I only see the path where I fork your repo (and the angular one) and make my own package for this. When I have Time, I will take this road, but I just wanted to check whether something like this is planned.

**Comment by acerix** [comment]
I believe this existing ticket captures the need to customize the toolbar:
https://github.com/fullcalendar/fullcalendar/issues/496
↑
Please visit the above issue, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates.

I can leave this open as a feature request for a new built-in theme using Angular Material.

**Comment by thomi137** [comment]
Hmmm. Maybe I was not quite clear in the heat of the moment. Sorry for that. I want to make an Angular8 (no JS) fc component which allows me to dynamically switch out the header (and/or footer) buttons, so I could make it look more like Material. Creating a custom view is not that difficult, so I got that part. The toolbar is not that open for extension, so that was my original request...

**Comment by acerix** [comment]
So do you mean you are looking for something like the "custom view via JS" feature but for the toolbar?

https://fullcalendar.io/docs/custom-view-with-js

**Comment by thomi137** [comment]
Exactly. I covered the custom view using that! Now I would very much like the same for the header. I found a workaround by disabling the header (set it to false) and making my own, using the API directly. I am for the moment quite satisfied with the result, but it requires a little tinkling with the Angular lifecycle hooks (e.g. if I want the title from the View object). Doable, but it feels a little dirty...
